cob_moveit_config/raw3-1/config/ompl_planning.yaml

```diff
@@ -1,75 +1,68 @@
 planner_configs:
```

Comment:
did you also update the SRDF?
you can use `rosrun cob_moveit_bringup update_collisions.sh raw3-1` to update the self-collision matrix
please also update the planning groups!